examples/with-react-native-web/apps/native/app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "expo": {
+    "name": "turborepo-native",
+    "slug": "turborepo-native",
```

Comment:
Let's drop our slug here; it's a guaranteed instant rename whereas native they may wait a week.

Reply:
😄 sounds good.